build/Dockerfile.operator

```diff
@@ -30,7 +30,7 @@ MAINTAINER IBM Storage
 ###Required Labels
 LABEL name="Operator for IBM block storage CSI driver" \
       vendor="IBM" \
-      version="1.0.0" \
+      version="1.1.0" \
       release="b16" \
```

Comment:
General question: what does the release build number (b16) mean ?

Reply:
@27149chen [should know](https://github.com/IBM/ibm-block-csi-operator/commit/6932f62dd3c2d45f3132d84fb98a92f628f62bad)

Reply:
It is a mandatory label for any redhat cetified images.
It should be generated in every build, needs improvement.

Reply:
created ticket CSI-1107